src/sdp.ts

```diff
@@ -80,14 +80,13 @@ t=0 0
 a=ice-lite
 m=application ${PORT} UDP/DTLS/SCTP webrtc-datachannel
 a=mid:0
-a=setup:active
-a=ice-options:ice2
+a=setup:passive
 a=ice-ufrag:${ufrag}
 a=ice-pwd:${ufrag}
 a=fingerprint:${CERTFP}
 a=sctp-port:5000
 a=max-message-size:100000
-a=candidate:1 1 UDP 1 ${IP} ${PORT} typ host`;
+a=candidate:1467250027 1 UDP 1467250027 ${IP} ${PORT} typ host\r\n`;
```

Comment:
where do these numbers come from, might be nice to add a line of comment about them?

Reply:
Not sure if a comment explaining is useful, but here is the grammar for the candidate line: https://www.rfc-editor.org/rfc/rfc8839.html#name-candidate-attribute . 

The foundation (first number) is a way to encode ice candidates. The RFC specifies what the foundation should do here: https://www.rfc-editor.org/rfc/rfc8445.html#section-5.1.1.3 , but in practice it is not used much. This can be any number for our purposes.

The second number is the stream component. `1` is RTP and `2` is RTCP. Since we multiplex RTP and RTCP over the same component, we use `1`. 

The third number (after UDP) is the candidate priority. Since we have only 1 candidate, this does not matter in this SDP.

Reply:
What I would do is make constants for this with very explicit declarative names, and you can link to the relevant paragraph in an RFC for each constant. I think that's enough documentation. Gives all the info with little effort.